.github/workflows/pull_request.yaml

```diff
@@ -26,6 +26,9 @@ jobs:
         with: 
           bundler-cache: true
 
+      - name: Select xcodes
+        run: xcodes select
+
```

Comment:
Same as [this](https://github.com/alphagov/govuk-mobile-ios-services/pull/3#discussion_r1579144399)